imagefit/conf.py

```diff
@@ -29,10 +29,11 @@ class ImagefitConf(AppConf):
     IMAGEFIT_CACHE_ENABLED = True
     IMAGEFIT_CACHE_BACKEND_NAME = 'imagefit'
 
-    settings.CACHES[IMAGEFIT_CACHE_BACKEND_NAME] = {
-        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
-        'LOCATION': os.path.join(tempfile.gettempdir(), 'django_imagefit')
-    }
+    # On Django1.8, this overwrites all CACHES?
+    #settings.CACHES[IMAGEFIT_CACHE_BACKEND_NAME] = {
+    #    'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
+    #    'LOCATION': os.path.join(tempfile.gettempdir(), 'django_imagefit')
+    #}
```

Comment:
Are you sure about removing these lines? This is supposed to be the default `IMAGEFIT_CACHE_BACKEND_NAME` that should be overridable in the settings.
Do you have any doc/info relating your choice?